aichallenge/workspace/src/aichallenge_submit/my_gnss_poser/config/gnss_poser.param.yaml

```
@@ -0,0 +1,38 @@
+# GNSS-IMU Poser Configuration Parameters
+# These parameters can be adjusted without rebuilding the code
+
+gnss_imu_poser:
+  ros__parameters:
+    # Basic configuration
+    ros_queue_size: 10
+    imu_history_max_age_sec: 2.0
+    imu_data_valid_age_sec: 0.1
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)